run_md.py

```diff
@@ -0,0 +1,879 @@
+                       how='after', n=n_line_after_molecules_section)
+
+
+def prep_ligand(mol, script_path, project_dir, wdir_ligand, addH=True):
+    molid = mol.GetProp('_Name')
+    resid = mol.GetProp('resid')
+
+    wdir_ligand_cur = os.path.join(wdir_ligand, molid)
+    os.makedirs(wdir_ligand_cur, exist_ok=True)
+
+    mol_file = os.path.join(wdir_ligand_cur, f'{molid}.mol')
+
+    if os.path.isfile(os.path.join(wdir_ligand_cur, f'{molid}.itp')) and os.path.isfile(os.path.join(wdir_ligand_cur, f'posre_{molid}.itp'))\
+            and os.path.isfile(os.path.join(wdir_ligand_cur, 'resid.txt')):
+        logging.warning(f'{molid}.itp and posre_{molid}.itp files already exist. Mol preparation step will be skipped for such molecule\n')
+        with open(os.path.join(wdir_ligand_cur, 'resid.txt')) as inp:
+            molid, resid = inp.read().strip().split('\t')
+
+        return wdir_ligand_cur, molid, resid
+
+    if addH:
+        mol = Chem.AddHs(mol, addCoords=True)
+
+    Chem.MolToMolFile(mol, mol_file)
+
+    try:
+        subprocess.check_output(f'script_path={script_path} lfile={mol_file} input_dirname={wdir_ligand_cur} name={resid} bash {os.path.join(project_dir, "lig_prep.sh")}',
+                                shell=True)
+    except subprocess.CalledProcessError as e:
+        logging.error(f'{molid}\t{e}\n')
+        return None
+
+    # create log for molid resid corresponding
+    with open(os.path.join(wdir_ligand_cur, 'resid.txt'), 'w') as out:
+        out.write(f'{molid}\t{resid}\n')
+    with open(os.path.join(wdir_ligand, 'all_resid.txt'), 'a') as out:
+        out.write(f'{molid}\t{resid}\n')
+
+    return wdir_ligand_cur, molid, resid
+
+def prep_md_files(var_lig_data, protein_name, system_lig_data, wdir_protein, wdir_md, clean_previous=False):
+    def copy_md_files_to_wdir(files, wdir_copy_to):
+        for file in files:
+            shutil.copy(file, os.path.join(wdir_copy_to, os.path.basename(file)))
+
+    if var_lig_data:
+        wdir_var_ligand_cur, var_lig_molid, var_lig_resid = var_lig_data
+        wdir_md_cur = os.path.join(wdir_md, f'{protein_name}_{var_lig_molid}')
+    else:
+        wdir_md_cur = os.path.join(wdir_md, protein_name)
+
+    if clean_previous and os.path.isdir(wdir_md_cur):
+        shutil.rmtree(wdir_md_cur)
+
+    os.makedirs(wdir_md_cur, exist_ok=True)
+
+    # topol for protein for all chains
+    copy_md_files_to_wdir(glob(os.path.join(wdir_protein,'*.itp')), wdir_copy_to=wdir_md_cur)
+    copy_md_files_to_wdir([os.path.join(wdir_protein, "topol.top")], wdir_copy_to=wdir_md_cur)
+
+    # prep ligands and cofactor
+    # copy ligand.itp to md_wdir_cur. Will be edited
+    if var_lig_data:
+        wdir_var_ligand_cur, var_lig_molid, var_lig_resid = var_lig_data
+        all_itp_list, all_gro_list, all_posres_list, all_lig_molids, all_resids = [os.path.join(wdir_md_cur, f'{var_lig_molid}.itp')],\
```

Comment:
split assignments on separate lines, one assignment - one line, it is more readable if there are a lot of variables